internal/database/external_services.go

```diff
@@ -1339,21 +1355,7 @@ ORDER BY es.id, essj.finished_at DESC
 		return nil, err
 	}
 
-	messages := make(map[int64]string)
-
-	for rows.Next() {
-		var svcID int64
-		var message sql.NullString
-		if err := rows.Scan(&svcID, &message); err != nil {
-			return nil, err
-		}
-		messages[svcID] = message.String
-	}
-	if err := rows.Err(); err != nil {
-		return nil, err
-	}
-
-	return messages, nil
+	return scanSyncErrors(rows, err)
```

Comment:
I'm 99% this works, saving you another 4 lines:

```suggestion
	return scanSyncErrors(e.Query)
```